packages/shared/lib/core/app/interfaces/platform-event-map.interface.ts

```diff
@@ -1,11 +1,8 @@
-import { DashboardRoute } from '@core/router'
-
 import { IAppUpdateDownloadProgress } from './app-update-download-progress.interface'
 import { IAppVersionDetails } from './app-version-details.interface'
 
 export interface IPlatformEventMap {
     'menu-logout': void
-    'menu-navigate-wallet': DashboardRoute
```

Comment:
We still use the 'menu-navigate-wallet' event in https://github.com/iotaledger/firefly/blob/55d2865ed49b36c01603286e58c731535cb62a82/packages/desktop/App.svelte#L89

I think we still need the event, but it can be of `void` type, which gets rid of the `DashboardRoute` dependency.